src/host/ledger.h

```diff
@@ -826,12 +849,15 @@ namespace asynchost
         {
           auto file_name = f.path().filename();
 
-          if (is_ledger_file_name_recovery(file_name))
+          if (
+            is_ledger_file_name_recovery(file_name) ||
+            is_ledger_file_name_ignored(file_name))
```

Comment:
Perhaps combine the two so that `is_ledger_file_name_ignored` also returns true when `is_ledger_file_name_recovery`?